tools/dm-query-status.md

```diff
@@ -0,0 +1,234 @@
+                    "result": null,         # Displays the error information if a subtask fails.
+                    "unresolvedDDLLockID": "test-`test`.`t_target`",    # The sharding DDL lock ID, used for manually handling the sharding DDL 
+                                                                        # lock in the abnormal condition.
+                    "sync": {                   # The synchronization information of the `Sync` processing unit. This information is about the 
+                                                # same component with the current processing unit.
+                        "totalEvents": "12",    # The total number of binlog events that are synchronized in this subtask.
+                        "totalTps": "1",        # The number of binlog events that are synchronized in this subtask per second.
+                        "recentTps": "1",       # The number of binlog events that are synchronized in this subtask in the last one second.
+                        "masterBinlog": "(bin.000001, 3234)",                               # The binlog position in the upstream database.
+                        "masterBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-14",    # The GTID information in the upstream database.
+                        "syncerBinlog": "(bin.000001, 2525)",                               # The position of the binlog that has been synchronized
+                                                                                            # in the `Sync` processing unit.
+                        "syncerBinlogGtid": "",                                             # It is always empty because `Sync` does not use GTID to
+                                                                                            # synchronize data.
+                        "blockingDDLs": [       # The DDL list that is blocked currently. It is not empty only when all the upstream tables of this 
+                                                # DM-worker are "synced". In this case, it indicates the sharding DDL statement to be executed or skipped.
+                            "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
+                        ],
+                        "unresolvedGroups": [   # The sharding group that is not resolved.
+                            {
+                                "target": "`test`.`t_target`",                  # The downstream database table to be synchronized.
+                                "DDLs": [
+                                    "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
+                                ],
+                                "firstPos": "(bin|000001.000001, 3130)",        # The starting position of the sharding DDL statement.
+                                "synced": [                                     # The sharded table of the sharding DDL statement that `Sync` has read.
```

Comment:
The shared table that `sync` unit has read its sharding DDL statement.
The original Chinese is not accuracy, `the shared table` does not belong to `sharding DDL`.

Reply:
The suggested version is still not appropriate in grammar. Please confirm the sentence logic with @amyangfei and provide a new version. :)